extensions/wikia/AdEngine/js/video/player/porvata/porvataPlayerFactory.js

```diff
@@ -56,6 +61,9 @@ define('ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory', ['wikia.l
 				ima.getAdsManager().resume();
 			},
 			setVolume: function (volume) {
+				if (videoAd) {
```

Comment:
it looks to me like we are using `videoAd` like a distinguish between mobile and desktop. Am I right?
(I'm basing this on this comment -> https://github.com/Wikia/app/pull/12203/files#diff-e77e2c1ef4d0877d80d59d95c1086bc9R37).